CodeForBoulder/upswyng
# Wrap Resource component with withResource HOC

This PR does not close an issue.

## What does this PR do?

- wraps `<Resource/>` with `<withResource/>` HOC.
- adds constants file to centralize variable values
- adds the `/resource` route

## Testing

1. Navigate to `/resource?id=[resource-id-here]` (example ID: `-KUnIgnkpl1PvoQzNOwU`)
2. See the name of the resource appear
3. Weep tears of joy

## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

![thats a wrap](https://media.giphy.com/media/8JZxSl9H3gsn9vtfYd/giphy.gif)
